tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs

```diff
@@ -56,5 +59,28 @@ public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags
             // or we couldn't populate it completely (some tags is better than no tags)
             return scope;
         }
+
+        public static string GetQueueName(string queueUrl)
+        {
+            if (queueUrl is null)
+            {
+                return queueUrl;
+            }
+
+            return queueUrl.Split('/').Last();
+        }
+
+        internal static string GetOperationName(Tracer tracer, string spanKind) => tracer.CurrentTraceSettings.Schema.Version switch
+        {
+            SchemaVersion.V0 => SqsRequestOperationName,
```

Comment:
I've seen the same issue in RabbitMQ integration. The current name doesn't match V0, so we reimplmeent in the integration, which is acceptable. 
I wonder though if we shouldn't add an extra `v0Override` parameter (by default null) in `GetOperationName` that we would use if not null or empty.

WDYT? also cc @pablomartinezbernardo as it applies for his PR as well. 
Also I'm ok for doing it in a separate PR afterwards (if it works). 

And while I'm at it, maybe we can add an extra `GetOperationName` method in `MessagingSchema` that would take the `spankind` as well in input to do the switch we do below.

Reply:
Thanks for the feedback Pablo. I'm good to leave this here. Anyway, it was not a feedback for the PR itself but more general to the current implem. Thanks

Reply:
I think this should be left inside each integration. Since it would be really complex in terms of readability. Since I think not every integration, that I have seen, follow this approach. Either way, I just started on this tracer, so feel free to dismiss that last sentence.

On the other hand, it seems that SNS is going to follow the same exact approach. So some code duplication might exist there. I think we should start worrying on it once multiple integrations showcase. the same symptoms where current name doesn't match V1.